primo/utils/solvers.py

```diff
@@ -66,10 +67,10 @@ def get_solver(
     ------
     ValueError
         If an unrecognized solver is provided as an input.
-        Supported solvers include highs, gurobi, scip, and glpk
+        Supported solvers include glpk, gurobi, gurobi_persistent, highs and scip
```

Comment:
Also, could you please set the default solver to `scip`? I did run into a few issues with `highs` on another problem. We will  still keep it as an optional solver though.

Reply:
works for me!